debezium-quarkus/poms/bom/pom.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
I'm struggling a bit to digest the number of POMs; I was familiar with runtime and deployment module when writing the Kafka Streams extensions. What are all the others (build-parent, bom, etc.) needed for?

Reply:
I'm going to consolidate these.  I mirrored a lot of my PoC from the camel-quarkus layout and these are artifacts of their build style which we really don't need.